### PR TITLE
Update DemoteUserCommand.php

### LINKED ISSUE #5585: [user] Wrong description for sylius:user:demote command
Message reads:

```
sylius:user:demote                               Promotes a user by adding a role.
```

### PR DESCRIPTION
**Pull request by gabiudrescu**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Related tickets | fixes #5585
| License         | MIT

### CODE REVIEW

**Comment by lchrusciel** [comment]
Does something go wrong with a rebase?

File: src/Sylius/Bundle/UserBundle/Command/DemoteUserCommand.php [hunk 1]
```diff
@@ -28,7 +28,7 @@ protected function configure()
     {
         $this
             ->setName('sylius:user:demote')
-            ->setDescription('Promotes a user by adding a role.')
+            ->setDescription('Demotes a user by removing a role.')
```
Comment (lchrusciel) [inline src/Sylius/Bundle/UserBundle/Command/DemoteUserCommand.php]: Is it all that should be changed?

Comment (gabiudrescu) [inline src/Sylius/Bundle/UserBundle/Command/DemoteUserCommand.php]: yup, it was a typo, from what I understood from #5585. 

to be honest, I haven't tested it locally, but I can do that if required. thow, this line `$user->removeRole($securityRole);` seems pretty clear in what it should do :)

**Comment by gabiudrescu** [comment]
@lchrusciel I didn't do a rebase. when I first did a PR, the build on master was failing. later last night, when the build was passing, I merged my branch with master just to see if it was passing after my small typo correction.

I assume I'll have to do a rebase to get this merged, right? :)

**Comment by lchrusciel** [comment]
It would be nice, if you would rebase it. But I guess removing all this commits and just cherry-picking a require one would be the fastest solution.

**Comment by gabiudrescu** [comment]
it was simpler for me just to redo everything. new PR here: #5619

**Pull request closed without merging**